pkgs/development/interpreters/python/default.nix

```diff
@@ -1,4 +1,4 @@
-{ pkgs, lib }:
+{ pkgs }:
 
 with pkgs;
 
```

Comment:
unnecessary, please revert

Reply:
Sorry, this is the reason why I wrote *”This PR now depends on #98185“* and removed the review request GitHub automatically dispatched. It is part of mentioned PR and I will rebase this PR once the dependency (now #98197) is merged.

What has it todo with openlp? Openlp uses pyqt_with_qtwebkit and the bug fixed in #98197 results in qt version 5.15 being used, for which qtwebkit is marked as broken.

Reply:
:(, idk. I guess you could help review other PR to help expedite it.

Reply:
The missing bits there are only FRidh's review and (probably) acceptance of my last patch.

In the future, I will isolate my PRs.